packages/sdk/src/controllers/ConnectorController.ts

```diff
@@ -197,7 +197,7 @@ export class ConnectorController {
      * @param id the id of your registered connector you want to make sure it is loaded
      * @returns
      */
-    waitToBeReady = async (id: Id, timeoutMilliseconds = 2000): Promise<EditorResponse<null>> => {
+    private waitToBeReady = async (id: Id, timeoutMilliseconds = 2000): Promise<EditorResponse<null>> => {

```

Comment:
in theory this is a breaking change

Reply:
I would say practically for those who are TypeScript based (We're btw, ) It's a breaking change (it's even breaking change for us in Studio UI)